scripts/bulk-password-update-export.js

```diff
@@ -67,6 +69,25 @@ Do you want to continue? [y/N]
   }));
 };
 
+const changeUserPass = async (user, password, options) => {
+  const admins = ['admin', 'medic'];
+  const adminError = new Error('403 - Cannot change password for "medic" or "admin" users.');
+  const postOptions = {...options};
+  postOptions.body = {
+    'password': password
+  };
+  postOptions.uri = `${options.uri}/${user}`;
+  try {
+    if (admins.includes(user)) {
+      throw adminError;
```

Comment:
Hmmm interesting. I think because you've created this in line 74 the stacktrace will show line 74 and not 82. I would just inline it here so it's created only if/when necessary (which feels like a tiny performance win too!), and also means you can include the specific bad username in the error message, eg:

```suggestion
      throw new Error(`403 - Cannot change password for administrator user "${user}"`);
```

Reply:
fixed in f0c5b2171 and c237a2f66